libbeat/docs/template-config.asciidoc

```diff
@@ -8,19 +8,22 @@ Elasticsearch.
 
 You can adjust the following settings to load your own template or overwrite an existing one:
 
-*`enabled`*:: Set to false to disable template loading. If set this to false,
+*`setup.template.enabled`*:: Set to false to disable template loading. If set this to false,
 you must <<load-template-manually,load the template manually>>.
 
-*`name`*:: The name of the template. The default is +{beatname_lc}+.
+*`setup.template.name`*:: The name of the template. The default is +{beatname_lc}+. The version of the Beat will always be appended to the given name
+so the final name is `filebeat-%{[beat.version]}`
 
-*`fields`*:: The path to the YAML file describing the fields. The default is +fields.yml+. If a
+*`setup.template.fields`*:: The path to the YAML file describing the fields. The default is +fields.yml+. If a
 relative path is set, it is considered relative to the config path. See the <<directory-layout>>
 section for details.
 
-*`overwrite`*:: A boolean that specifies whether to overwrite the existing template. The default
+*`setup.template.overwrite`*:: A boolean that specifies whether to overwrite the existing template. The default
 is false.
 
-*`settings.index`*:: A dictionary of settings to place into the `settings.index` dictionary of the
+//REVIEWERS: I don't see the following options in the reference.yml file. Are they still available/valid?
```

Comment:
The `setup.template.settings` shows up only in the metricbeat.yml, not in the reference.yml. I will fix it.